docs/API.md

```diff
@@ -430,7 +431,7 @@ Parameters.
 Each token takes the following fields for it's create method respectively:
 
 * sessionToken : data, uid, createdAt, uaBrowser, uaBrowserVersion, uaOS, uaOSVersion, uaDeviceType,
-                 uaFormFactor, mustVerify, tokenVerificationId
+                 uaFormFactor, mustVerify, tokenVerificationId, shortCode, shortCodeExpiresAt
```

Comment:
naming nit: I wonder if we should include the word "verification" somewhere in the name of this thing, to make it more clear what it's for.

Reply:
(that may be less of a concern if it's only for internal use by this repo, and is return returned to the auth-server)